### ISSUE TITLE
ExclusionStrategy equivalent? 

### ISSUE THREAD

**Issue by SeanOThomas** [cbeust/klaxon#136]
Does Klaxon have an equivalent feature to Gson's `ExclusionStrategy ` for dynamically ignoring property fields?

**Comment by cbeust** [comment]
Yes, either with `@Json(ignored = true)` or simply by declaring that property `private`.

**Comment by SeanOThomas** [comment]
Sorry, I mean, how do you ignore fields dynamically? That's essentially what an `ExclusionStrategy` achieves. Changing the value of an annotation dynamically seems clunky.

**Comment by cbeust** [comment]
Oh sorry, I misunderstood.

Not supported at the moment but that's easy enough to add. I'm guessing that when Klaxon is about to process a field, it would invoke some listener that would respond whether to ignore that field or not?

**Comment by SeanOThomas** [comment]
That makes sense. For now, I supposed I could remove the properties from `value` in `toJson()`. before serializing.

**Comment by SeanOThomas** [comment]
Do you anticipate adding an `ExclusionStrategy` feature? If so, do you have an approximate idea of when? 

Thanks so much!

**Comment by cbeust** [comment]
First pass available [here](https://github.com/cbeust/klaxon/commit/aea61af1594a3f00e8e85aa88c1adad152a37691).

The interface name is `PropertyStrategy`, take a look at `PropertyStrategyTest` and let me know what you think.

**Comment by SeanOThomas** [comment]
Awesome! Looks good. Thank you for the quick turnaround.
